CONTRIBUTING.md

```diff
@@ -177,6 +177,32 @@ which shows the baseline image, the generated image, the diff and the json mocks
 
 To view the results of a run on CircleCI, download the `build/test_images/` and `build/test_images_diff/` artifacts into your local repo and then run `npm run start-image_viewer`.
 
+### Writing selection tests
```

Comment:
Thanks for doing this!

Your tips don't just apply to selection tests though. Hover and click tests also rely on these relative-top-left pixel coordinates. Maybe this section header could instead be _Writing interaction tests_?